src/utils/index.ts

```diff
@@ -194,3 +194,26 @@ export function isPrivateKey(string: string): boolean {
 
   return /^[a-f0-9]{64}$/.test(normalized)
 }
+
+export function toFixed(x: number): string {
+  let res = '0'
+
+  if (Math.abs(x) < 1.0) {
+    const e = parseInt(x.toString().split('e-')[1])
+
+    if (e) {
+      x *= Math.pow(10, e - 1)
+      res = '0.' + new Array(e).join('0') + x.toString().substring(2)
+    }
+  } else {
+    let e = parseInt(x.toString().split('+')[1])
+
+    if (e > 20) {
+      e -= 20
+      x /= Math.pow(10, e)
+      res += new Array(e + 1).join('0')
+    }
+  }
+
+  return res
+}
```

Comment:
I am not sure if this function is working. I was testing it like this (proper unit tests would be better):

```
console.log(toFixed(1000))
console.log(toFixed(10))
console.log(toFixed(1))
console.log(toFixed(0.1))
console.log(toFixed(0.01))
console.log(toFixed(0.000001))
console.log(toFixed(0.000000000000001))
```

Output:

```
0
0
0
0
0
0
0.000000000000001
```

Reply:
Yes, it was intended solely for decimals. I completely removed it

Reply:
Nice, thank you.

If `toFixed` is dead code please remove it